public/index.html

```diff
@@ -15,7 +15,7 @@
   </div>
   <body>
     <div id="app"></div>
-
+    <a  class = 'login' href="/login/facebook" target="_self">Log In with Facebook</a>
```

Comment:
It is hard to see this button.

You could fix this by altering styles.css to add a case for .login:
```css
.login {
  position: relative;
  z-index: 1;
}
```

This should fix the issue, but additional styling may be necessary to make it look good.